django/src/rdwatch/views/region.py

```diff
@@ -1,13 +1,39 @@
-from rest_framework import viewsets
-from rest_framework.pagination import PageNumberPagination
+import iso3166
+from ninja import Schema
+from ninja.pagination import RouterPaginated
+
+from django.http import HttpRequest
+from django.shortcuts import get_object_or_404
 
 from rdwatch.models import Region
-from rdwatch.serializers import RegionSerializer
 
 
-class RegionViewSet(viewsets.ReadOnlyModelViewSet):
-    queryset = Region.objects.all().select_related('classification')
-    serializer_class = RegionSerializer
-    pagination_class = PageNumberPagination
-    # Set the number of items per page to 1000
-    pagination_class.page_size = 1000
+class RegionSchema(Schema):
+    id: int
+    name: str
+
+    @staticmethod
+    def resolve_name(obj: Region) -> str:
+        country = obj.country
+        classification: str = obj.classification.slug
+        number = obj.number
+
+        country_numeric = str(country).zfill(3)
+        country_code = iso3166.countries_by_numeric[country_numeric].alpha2
+        region_number = 'xxx' if number is None else str(number).zfill(3)
+        return f'{country_code}_{classification}{region_number}'
+
+
+router = RouterPaginated()
```

Comment:
wasn't an issue with performers but we have ~60 regions so the limit on pages for regions by default I think is 100.  Just so we know if we start hitting a high number of regions.